### PR TITLE
Bump versions according to dependabot alert

### PR DESCRIPTION
**Pull request by tjleing**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -19,8 +19,8 @@ gem "jekyll", "~> 3.6.2"
 
 # If you have any plugins, put them here!
 group :jekyll_plugins do
-  gem "jekyll-feed", "~> 0.6"
-  gem "github-pages"
+  gem "jekyll-feed"
+  gem "github-pages", "~> 209"
```
Comment (sammartinez) [inline Gemfile:23]: why this change?

Comment (tjleing) [reply Gemfile:23]: `github-pages` and `jekyll` were refusing to update until both packages had minimum versions, because they both depended on each other.

Comment (tjleing) [reply Gemfile:23]: Maybe now that they've been updated, we could remove this?  Although I feel like we'd have the same problem again.

Comment (sammartinez) [reply Gemfile:23]: Nah that makes sense so we can go ahead and leave them

**Review by sammartinez — APPROVED** [review]
LGTM 🌮
